features/step_definitions/basic_steps.rb

```diff
@@ -19,4 +19,15 @@
 
 When("I fill in {string} with {string}") do |field, input|
     fill_in field, with: input
+end
+
+When("I click on {string} on {string}") do |element, product_name|
+  product = Product.find_by_name(product_name)
+  within("#product_#{product.id}") do
+    click_on element
+  end
+end
+
+Then("stop") do
+  binding.pry
```

Comment:
remove this step

Reply:
done!

Reply:
They can keep steps like that for future use @faraznaeem No need to remove that definition

Reply:
It's already deleted. We can add it again in a later PR.